### PR TITLE
feature/SOF-4353 - Create documentation

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: lang/en/docs/properties-directory/non-scalar/file-content.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# File Content
+
+<span class="btn badge b-success border-50">Non-Scalar</span> <span class="btn badge b-info border-50">Electronic</span>
```
Comment (timurbazhirov) [inline lang/en/docs/properties-directory/non-scalar/file-content.md:3]: File content is not an electronic property

File: lang/en/docs/workflows/data/units.md [hunk 2]
```diff
@@ -44,6 +45,36 @@ For a description of unit input templating, the reader is referred to [this sect
 {!example/workflow/unit/io.json!}
 ```
 
+### API (Input Section Only)
```
Comment (timurbazhirov) [inline lang/en/docs/workflows/data/units.md:48]: We don't need to specify "Input Section Only"

File: lang/en/docs/properties-directory/non-scalar/file-content.md (hunk 1 above)
Comment (AcylSilane) [reply lang/en/docs/properties-directory/non-scalar/file-content.md:3]: Fixed

File: lang/en/docs/workflows/data/units.md (hunk 2 above)
Comment (AcylSilane) [reply lang/en/docs/workflows/data/units.md:48]: Removed

File: lang/en/docs/models-directory/machine-learning/accuracy.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+When training a machine learning model, care must be taken to ensure that the dataset being used is a quality one.
+Indeed, the concept of "garbage-in, garbage-out" is highly applicable to models trained on poor a low-quality dataset -
+a model is only as good as the data it was trained on.
+
+Consider a hypothetical machine learning model which is able to achieve perfect accuracy on any dataset it's given,
+without overfitting. Now consider what would happen if such a model were trained on two different datasets of vastly
+different quality - for example, electronic energies calculated using
+the [Local-Density Approximation](https://en.wikipedia.org/wiki/Local-density_approximation) versus electronic energies
+calculated with a [Coupled-Cluster](https://en.wikipedia.org/wiki/Coupled_cluster) technique. Despite performing
```
Comment (timurbazhirov) [inline lang/en/docs/models-directory/machine-learning/accuracy.md:13]: Lets not use a direct link but instead add a links section below like in a manuscript

File: lang/en/docs/models-directory/machine-learning/example-workflow.md [hunk 4]
```diff
@@ -2,24 +2,1692 @@
 
 A Machine Learning [workflow](../../workflows/overview.md) may consist of the following multiple steps, performed in a sequential order through the [units](../../workflows/components/units.md) specific to Machine Learning which are described [in this page](units.md).
 
-## Training a Model
+Below, we show an example schema for both the training and prediction version of a workflow.
```
Comment (timurbazhirov) [inline lang/en/docs/models-directory/machine-learning/example-workflow.md:5]: schema => implementation

Comment (AcylSilane) [reply lang/en/docs/models-directory/machine-learning/example-workflow.md]: Adjusted

File: lang/en/docs/models-directory/machine-learning/accuracy.md (hunk 3 above)
Comment (AcylSilane) [reply lang/en/docs/models-directory/machine-learning/accuracy.md]: Links section added

**Pull request merged**